packages/blockchain-extension/resources/tutorials/developer-tutorials/privateData.md

```diff
@@ -90,11 +88,13 @@ The `verifyMyPrivateAsset` transaction can be used to verify if a given set of p
 
 Notice the lines that start with `@Transaction()`. These come before each function in the generated `Private Data Contract` and they define the preceding transaction to be a callable transaction function. `@Transaction()` takes a boolean parameter where `true` indicates that the function is intended to be called using  `submit` and `false` indicates that the function is intended to be called using `evaluate`, with the default being set to true. The differences between `@Transaction()` and `@Transaction(false)` is especially important when it comes to the `readMyPrivateAsset()` transaction. A `Submit transaction` MUST NOT be performed on the `readMyPrivateAsset()`transaction. Doing so would result in the returned private data being submitted to the public ledger.
 
-It is also worth having a look at the collections.json file at the root of the file directory for your project. Collections.json is the file that defines information such as who can persist data, how long private data is stored in a private database and the number of peers required for disseminating data – as well as how many it can be distributed to. Notice that the MSP ID is the one you specified when creating the project – whichever org is specified in the MSP ID is the one who can perform transactions.
+At the top of the `Private Data Contract` you'll see the function `getCollectionName`, a helper function that returns the `collectionName` that is required when transacting with a private contract. It is currently implemented to utilize the implicit collection that is created when you create a Local Fabric network through the extension. Implicit collections are a v2.0 Hyperledger feature, find out more about them [on the official Hyperledger Fabric docs](https://hyperledger-fabric.readthedocs.io/en/release-2.2/private-data-arch.html#referencing-implicit-collections-from-chaincode).
```

Comment:
Not sure we need:
`It is currently implemented to utilize the implicit collection that is created when you create a Local Fabric network through the extension.`

Amend to `v2.0 Hyperledger Fabric feature`